plugins/com.google.cloud.tools.eclipse.dataflow.core/.project

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>com.google.cloud.tools.eclipse.dataflow.core</name>
+	<comment></comment>
+	<projects>
+	</projects>
```

Comment:
OSGi/the PDE/Eclipse/something will automatically figure out our dependencies, I imagine?